python/lsst/ts/m2gui/controltab/tab_detailed_force.py

```diff
@@ -193,6 +200,24 @@ def _resize_table(self, table):
 
         return table
 
+    @asyncSlot()
+    async def _callback_time_out(self):
+        """Callback timeout function to update the force table."""
+
+        # Update the force items on tables
+        for key in vars(self._forces_latest).keys():
+            field = getattr(self._forces_latest, key)
+
+            is_interger = key in ("encoder_count", "step")
+
+            for idx in range(NUM_ACTUATOR):
+                if is_interger:
+                    self._forces[key][idx].setText(str(field[idx]))
+                else:
+                    self._forces[key][idx].setText("%.2f" % field[idx])
+
+        self.check_duration_and_restart_timer(self._timer)
```

Comment:
If that would be switched to timerEvent (see above), then perhaps keep last timer interval, check if the model interval matches settings, and if mis-matched, then killTimer(event.timerId()) and start a new timer.